modules/xml/src/main/resources/META-INF/mule-xml.xsd

```diff
@@ -514,14 +514,25 @@ The Jaxen filter allows you to route messages based on XPath expressions. The Ja
                     </xsd:element>
                     <xsd:element name="context-property" type="spring:entryType" minOccurs="0" maxOccurs="unbounded">
                         <xsd:annotation>
-                            <xsd:documentation>A property that wil be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
+                            <xsd:documentation>A property that will be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
                 </xsd:sequence>
+                <xsd:attribute name="timeBetweenTransformerEvictions" type="mule:substitutableInt">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                          Transformers are pooled for better throughput, since performing an XSL
```

Comment:
The only thing I want is #2, not #1. I don't think that I've stated a desire for prepopulation, but please point it out to me if I mistakenly gave that impression.

Nevertheless, the question of whether or not eviction is happening with this changeset needs to be ironed out and that can and should be done by a test. I will write a test for eviction and either (1) add it to the changeset if it proves that the changeset allows for eviction or (2) fix the changeset to allow for eviction if the test proves the changeset doesn't do so already. This is my mistake for not including such a test from the get-go and I will rectify it. Once complete I will push the changes for further review.